requirements.txt

```diff
@@ -0,0 +1 @@
+Flask==1.1.1
```

Comment:
why not add flake8 and pytest to this file instead of installing them individually.

Reply:
because flake8 and pytest are not required in a DEV, STAGE and PROD environments.

Reply:
Is this the requirements file for the CI or for the whole project? 
Flask is currently installed in setup.sh without using a requirements file

Reply:
yes, we decided that we used the requirement file